eng/testing/ildescriptors/ILLink.Descriptor.Serialization.xml

```diff
@@ -0,0 +1,10 @@
+<linker>
+  <assembly fullname="System.Private.DataContractSerialization">
+    <type fullname="System.Runtime.Serialization.KeyValue`2" />
```

Comment:
cc @joperezr - is your change going to make it so this is no longer necessary?

Reply:
In general, I don't think we should be adding `ILLink.Descriptors` files for our own assemblies. Instead we should be fixing our assemblies to be trimmable.